src/spotify_to_tidal/sync.py

```diff
@@ -224,12 +224,20 @@ def get_tracks_for_new_tidal_playlist(spotify_tracks: Sequence[t_spotify.Spotify
     ''' gets list of corresponding tidal track ids for each spotify track, ignoring duplicates '''
     output = []
     seen_tracks = set()
+    duplicate_count = 0
```

Comment:
This is a bit noisy for my liking tbh, I'd rather not have the duplicate count printed for every playlist. I think it's enough to just print a message for each duplicate, the user can go through the log and count them if they want to. I should hope it's a relatively rare occurence.